h/db/__init__.py

```diff
@@ -136,6 +137,25 @@ def _maybe_create_world_group(engine, authority):
     session.close()
 
 
+def _maybe_create_world_organization(engine, authority):
+    from h import models
+    session = Session(bind=engine)
+    world_org = session.query(models.Organization).filter_by(pubid='__world__').one_or_none()
+    if world_org is None:
+        world_org = models.Organization(name=u'Hypothesis',
+                                        authority=authority,
+                                        pubid = '__world__',
+                                        )
+        with open(r'h/static/images/icons/logo.svg', 'r') as h_logo:
+            # Skip the first line in the file as it's the xml version info.
+            h_logo.readline()
```

Comment:
You shouldn't need to skip the first line here. The XML declaration tag is a proper part of the XML document and both the browser and `ElementTree` should accept it.